src/main/java/FeatureExtraction/ExtractionAlgorithm.java

```diff
@@ -0,0 +1,7 @@
+package FeatureExtraction;
```

Comment:
To raczej nie jest zalecana nazwa pakietu (CamelCase). Ustaliliśmy, że będzie to w postaci tletters.nazwapakietu.